### ISSUE TITLE
Server World Saves

### ISSUE THREAD

**Issue by Dizzyswirl** [AlexProgrammerDE/balena-minecraft-server#24]
Where does the server save the world data? I would like to be able to backup my worlds and know where to save them if i wanted to change worlds from config file. I assume this can be done?

**Comment by AlexProgrammerDE** [comment]
@Dizzyswirl yes thats possible. You just need to connect to the Pi via SCP and then to go into the serverfiles directory. You will find there three folders: world, world_nether and world_the_end

**Comment by Dizzyswirl** [comment]
I ended up finding those folders if I wanted to backup one of the current worlds, however would I be able to import a save from a win10 version of MC realms? I downloaded the realm save and it includes level.dat, a folder labeled "db" with .ldb files and some other files in root such as levelname.txt and level.dat_old. I'm wondering if it would be possible to import these into my pi server via SCP, any feedback is appreciated!
